### PR TITLE
broken link fixed w/signoff

### LINKED ISSUE #9957: [Docs] Broken Link
### Current State
Many of the links are still broken in this page  (https://docs.meshery.io/guides/configuration-management/working-with-designs)

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by prathyushmadhu**
**Notes for Reviewers**
previous PR on ImageHub didn't solve the issue. I have fixed three broken link issues in "https://docs.meshery.io/guides/configuration-management/working-with-designs"
1) ImageHub link
2) sample applications link
3) Performance Management with meshery link

This PR fixes #9957 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 992f77298baacc52889ba5b9316d20531c08dd64 at: https://65a39566f7a4795005ad9bc9--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Hi @prathyushmadhu   ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Like patterns, Meshery also comes with some sample WebAssembly Filters for you t
 
 You can also import these filters manually to your provider from the [wasm-filters](https://github.com/layer5io/wasm-filters) repo.
 
-Meshery's sample application [ImageHub](guides/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
+Meshery's sample application [ImageHub](/docs/pages/guides/infrastructure-management/deploying-sample-apps.md) will let you test out configuring these filters out-of-the-box.
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/configuration-management.md:77]:
```suggestion
Meshery's sample application [ImageHub]({{ site.baseurl }}/guides/infrastructure-management/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
```

@prathyushmadhu please accept and use this change.

Don't include filename extension.
Don't include "/docs".
Do use `{{ site.baseurl }}`.

**Review by leecalcote — CHANGES_REQUESTED** [review]
See the feedback provided. Apply this to all links.

File: docs/pages/guides/configuration-management/configuration-management.md (hunk 1 above)
Comment (prathyushmadhu) [reply docs/pages/guides/configuration-management/configuration-management.md]: @leecalcote I have made the required changes and have commited to the same PR.

**Comment by l5io** [comment]
🚀 Preview for commit 2ff4c1d7d573e7a81212d66892da51507cc6ff28 at: https://65a4a308e7ac41ee91c30c6f--meshery-docs-preview.netlify.app

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
@prathyushmadhu The links are still broken.
Screenshot 2024-01-15 at 11 10 15 PM
Screenshot 2024-01-15 at 11 10 02 PM

**Comment by prathyushmadhu** [comment]
@MUzairS15 I will look into that and will get to you.

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Meshery can also manage your Kubernetes applications and deploy them to any of y
 
 ### Managing Applications Through Meshery UI
 
-Meshery has a set of [sample applications](./deploying-sample-apps.md) which you can use to quickly test out your deployment.
+Meshery has a set of [sample applications]({{ site.baseurl }}/pages/guides/infrastructure-management/deploying-sample-apps) which you can use to quickly test out your deployment.
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/configuration-management.md:107]:
```suggestion
Meshery has a set of [sample applications]({{ site.baseurl }}/guides/infrastructure-management/deploying-sample-apps) which you can use to quickly test out your deployment.
```

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ Like patterns, Meshery also comes with some sample WebAssembly Filters for you t
 
 You can also import these filters manually to your provider from the [wasm-filters](https://github.com/layer5io/wasm-filters) repo.
 
-Meshery's sample application [ImageHub](guides/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
+Meshery's sample application [ImageHub]({{ site.baseurl }}/pages/guides/infrastructure-management/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/configuration-management.md:77]:
```suggestion
Meshery's sample application [ImageHub]({{ site.baseurl }}/guides/infrastructure-management/deploying-sample-apps) will let you test out configuring these filters out-of-the-box.
```

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 4]
```diff
@@ -118,7 +118,7 @@ The application files can be described in following formats:
 - Helm charts
 - Docker Compose Apps
 
-You can also run the same application on multiple connected kubernetes clusters and compare the performance. See [Performance Management with Meshery](performance-management.md) for more details.
+You can also run the same application on multiple connected kubernetes clusters and compare the performance. See [Performance Management with Meshery]({{ site.baseurl }}/pages/guides/performance-management/performance-management) for more details.
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/configuration-management.md:121]:
```suggestion
You can also run the same application on multiple connected kubernetes clusters and compare the performance. See [Performance Management with Meshery]({{ site.baseurl }}/guides/performance-management/performance-management) for more details.
```

**Review by leecalcote — APPROVED** [review]
THanks!

**Pull request merged**